packages/esm-appointments-app/src/calendar/monthly/monthly-workload-view.component.tsx

```diff
@@ -61,7 +61,7 @@ const MonthlyWorkloadView: React.FC<MonthlyWorkloadViewProps> = ({ dateTime, eve
       )}>
       {isSameMonth(dateTime, dayjs(selectedDate)) && (
         <p>
-          <div className={classNames(styles.totals)}>
+          <div className={classNames(styles.totals)} onClick={() => onClick('')}>
```

Comment:
interesting... how does this work @usamaidrsk ?

Reply:
```typescript
const onClick = (serviceUuid: string) => {
    navigate({ to: `${spaHomePage}/appointments/${dayjs(dateTime).format('YYYY-MM-DD')}/${serviceUuid}` });
  };
``` 

This here is the function definition @mogoodrich .
Which is also used on here.
```typescript
{visibleServices.map(({ serviceName, serviceUuid, count }, i) => (
                <div
                  key={`${serviceUuid}-${count}-${i}`}
                  role="button"
                  tabIndex={0}
                  onClick={(e) => {
                    e.stopPropagation();
                    onClick(serviceUuid);
                  }}
                  className={styles.serviceArea}>
                  <span>{serviceName}</span>
                  <span>{count}</span>
                </div>
              ))}
```

Reply:
Maybe I need to rename the function to be more meaningful from what was there 
Thanks

Reply:
Oh, nevermind, I get it now, thanks!